lib/cisco_node_utils/cmd_ref/vrf.yaml

```diff
@@ -41,6 +41,25 @@ destroy:
   ios_xr:
     set_value: 'no vrf <vrf>'
 
+mhost_ipv4:
```

Comment:
I thought there are 2 different CLI commands, which need to be 2 items in the YAML file. Is it true?